src/sbin/sympa.pl.in

```diff
@@ -472,11 +473,23 @@ if ($main::options{'dump'} or $main::options{'dump_users'}) {
         printf $fh "#\n#\tAliases for all Sympa lists open on %s\n#\n",
             $robot;
         close $fh;
+
+        my $aliases;
+
+        if ($alias_manager eq 'Template') {
+            $aliases = Sympa::Aliases->new('Template', file => $sympa_aliases);
+        }
+
         foreach my $list (@{$all_lists || []}) {
             next unless $list->{'admin'}{'status'} eq 'open';
 
-            system($alias_manager, 'add', $list->{'name'}, $list->{'domain'},
-                $sympa_aliases);
+            if ($alias_manager eq 'Template') {
+                $aliases->add($list);
+            }
+            else {
+                system($alias_manager, 'add', $list->{'name'}, $list->{'domain'},
+                       $sympa_aliases);
+            }
```

Comment:
I realized that `--make_alias_file` option should be effective only if the value of `alias_manager` is either "`Template`" or `$SBINDIR/alias_manager.pl`, and in both cases `Sympa::Aliases::Template` may be used.
In the other cases (`none`, `CheckSMTP` or arbitrary program) alias files won't be generated.

Reply:
OK, that makes sense to me. Should I adjust the PR accordingly?

Reply:
Yes, please. Thanks!

Reply:
I submitted #1084